lib/src/widgets/chat/ds_image_message_bubble.widget.dart

```diff
@@ -102,64 +104,112 @@ class DSImageMessageBubble extends StatelessWidget {
+                  onTap: () {
+                    if (!_controller.error.value) {
+                      _controller.appBarVisible.value = false;
+                      showGeneralDialog(
+                        context: context,
+                        barrierDismissible: false,
+                        transitionDuration: DSUtils.defaultAnimationDuration,
+                        transitionBuilder: (_, animation, __, child) =>
+                            _buildTransition(animation, child),
+                        pageBuilder: (context, _, __) => _buildPage(context),
+                      );
+                    }
+                  },
+                  child: Container(
+                    constraints: const BoxConstraints(
+                      maxHeight: 480.0,
+                    ),
+                    child: DSCachedNetworkImageView(
+                      fit: BoxFit.cover,
+                      width: width,
+                      url: url,
+                      placeholder: (_, __) => const Padding(
+                        padding: EdgeInsets.all(80.0),
+                        child: CircularProgressIndicator(),
+                      ),
+                      onError: _controller.setError,
+                    ),
                   ),
-                  DSBodyText(
-                    text: imageText!,
-                    color: align == DSAlign.right
-                        ? DSColors.neutralLightSnow
-                        : DSColors.neutralDarkCity,
+                ),
+                SizedBox(
+                  width: width,
+                  child: Padding(
+                    padding: const EdgeInsets.all(12.0),
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        DSCaptionText(
+                          text: imageTitle,
+                          color: align == DSAlign.right
+                              ? DSColors.neutralLightSnow
+                              : DSColors.neutralDarkCity,
+                        ),
+                        if (imageText != null) ...[
+                          const SizedBox(
+                            height: 6.0,
+                          ),
+                          DSBodyText(
+                            overflow: TextOverflow.clip,
+                            text: imageText!,
+                            color: align == DSAlign.right
+                                ? DSColors.neutralLightSnow
+                                : DSColors.neutralDarkCity,
+                          ),
+                        ],
+                      ],
+                    ),
                   ),
-                ]
+                ),
               ],
-            ),
-          ),
-        ],
+            );
+          }
+          return const SizedBox();
+        },
       ),
     );
   }
+
+  Future<ImageInfo> getImageInfo(final String url) async {
```

Comment:
If makes sense, this ```getImageInfo``` method could be placed in the controller file.

Reply:
Done